package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.10.0-pre",
   "description": "Fast math typesetting for the web.",
   "main": "dist/katex.js",
-  "browser": "dist/katex.min.js",
```

Comment:
Why was this reverted?

Reply:
@kevinbarabash #1057 is no longer needed, since webpack doesn't uses name `require` and https://github.com/browserify/browserify/issues/374 is not an issue.